### PR TITLE
manifest updates for dependabot prs

### PR DESCRIPTION
**Pull request by petrblaho**
- Adds name and uses token in check image GH action
- Adds name and condition to not run Dependabot's actions
- Adds GH action to update manifest on Depedabot's PRs

DRFT-1345

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by petrblaho** [comment]
needs #321 merged first to solve image build issue

File: .github/workflows/update_manifest_dependabot_pr.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      with:
+        python-version: '3.9'
+        cache: 'poetry'
+    - name: Install dependencies
+      run: poetry install --with dev --verbose
+    - name: Generate manifest
+      run: poetry run ./scripts/create-manifest.py
+    - name: Configure git
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```
Comment (vkrizan) [inline .github/workflows/update_manifest_dependabot_pr.yml:38]: magic number?

**Review by vkrizan — APPROVED** [review]

File: .github/workflows/update_manifest_dependabot_pr.yml (hunk 1 above)
Comment (petrblaho) [reply .github/workflows/update_manifest_dependabot_pr.yml:38]: yes... https://github.com/orgs/community/discussions/26560

**Pull request merged**